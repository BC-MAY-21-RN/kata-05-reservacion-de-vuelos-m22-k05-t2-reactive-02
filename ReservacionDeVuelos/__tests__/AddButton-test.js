import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import AddButton from '../src/components/atoms/AddButton';

describe('Tests from AddButton component', () => {
  it('AddButton snapshot test', () => {
    const renderAddButton = renderer.create(<AddButton />).toJSON();
    expect(renderAddButton).toMatchSnapshot();
  });
});
