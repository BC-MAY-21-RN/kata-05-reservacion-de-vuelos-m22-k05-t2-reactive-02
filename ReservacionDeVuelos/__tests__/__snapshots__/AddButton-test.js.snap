// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests from AddButton component AddButton snapshot 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "#5C6DF8",
      "borderRadius": 60,
      "bottom": 20,
      "flex": 1,
      "height": 69,
      "justifyContent": "center",
      "opacity": 1,
      "position": "absolute",
      "width": 69,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "center",
        "color": "#ffffff",
        "fontSize": 50,
        "paddingBottom": 6,
      }
    }
  >
    +
  </Text>
</View>
`;
